spec/support/factory_girl.rb

```diff
@@ -11,13 +11,17 @@
   end
 
   factory :list do
-    sequence(:title) { |n| "This is awesome #{n}" }
+    sequence(:title) { |n| "Groceries #{n}" }
+    due_date 'string'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.